hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/metrics/DataNodeMetricHelper.java

```diff
@@ -72,8 +71,10 @@ public static void getMetrics(MetricsCollector collector,
         "blocks failed to cache"), beanClass.getNumBlocksFailedToCache())
       .addGauge(Interns.info("NumBlocksFailedToUnCache", "Datanode number of" +
           " blocks failed in cache eviction"),
-        beanClass.getNumBlocksFailedToUncache());
-
+        beanClass.getNumBlocksFailedToUncache())
+      .addGauge(Interns.info("LastDirectoryScannerFinishTime",
+          "Last finish time of directory scanner"),
```

Comment:
"Last finish time of directory scanner" -> "Finish time of the last directory **scan**".

I would actually prefer "most recent" over "last", but fine with "last".

Reply:
@xinglin Sir, Thanks a lot for reviewing. modify "Last finish time of directory scanner" -> "Finish time of the last directory scan".  "last" reserved.